### DISCUSSION TITLE
Graphic Marks toolbar

### DISCUSSION THREAD

**Discussion by dylansproule** [klinecharts/KLineChart#141] [Q&A]
Hello, I was checking out the demo at https://liihuu.github.io/KLineChart/ and loved the integrated graphicMarks toolbar.

Do you have the toolbar in a vue component as an example to share?

Thanks!

**Comment by liihuu** [comment]
The sample page code is not public.

**Reply by dylansproule** [reply]
No problem, thanks the the reply.

**Reply by Maks-KaNDeR1** [reply]
It’s not a pancake, and here it’s a pancake, it would be better if you threw off the code where there is a toolbar and helped, what to do, the fucking assistant
